docs/dev/core/logging-levels.md

```diff
@@ -0,0 +1,35 @@
+# Logging
+
+On the Glean SDK's Rust crates we use the [log](https://github.com/rust-lang/log) crate
+to display log messages.
+
+## Logging Levels
+
+### [trace](https://docs.rs/log/0.4.11/log/macro.info.html)
+
+- Used for logging very granular information about the flow of glean-core internals. This logging level is meant for helping with bug diagnostics and may be very noisy.
+    - **Example**: `log::trace!("{} pings left in the queue (only deletion-request expected)", queue.len());`
+
+### [debug](https://docs.rs/log/0.4.11/log/macro.debug.html)
+
+- Used for logging messages that contain internal glean-core information. This logging level is meant for helping with bug diagnostics and may be very noisy.
+    - **Example**: `log::debug!("Database path: {:?}", path.display());`
+
+### [info](https://docs.rs/log/0.4.11/log/macro.info.html)
+
+- Used for logging informational messages. These messages should be enough for glean-core developers and users to verify that Glean is working as expected during development.
+    - **Example**: `log::info!("Ping {} successfully sent {}.", document_id, status);`
+
+### [warn](https://docs.rs/log/0.4.11/log/macro.warn.html)
+
+- Used for logging internal glean-core errors. These errors do not require further action of a Glean consumer, but may be informative to diagnose bugs.
```

Comment:
I think I would rephrase that to make it clear these are not "user-actionable" errors, but really warnings
Maybe:

Used for logging internal glean-core warning. These could be internal errors or warnings, that do not require further action by a Glean consumer, but may be informative to diagnose bugs.